sdk/core/core/src/az_http_policy_logging.c

```diff
@@ -58,6 +58,8 @@ static az_result _az_http_policy_logging_append_http_request_msg(
     _az_http_request const* request,
     az_span* ref_log_msg)
 {
+  extern az_span const _az_auth_header_name;
```

Comment:
This link between `az_http_policy_logging.c` and `az_credential_client_secret.c` is unfortunate. Is there any way to decouple them?

Also, do we know if "authorization" is the only header we need to avoid logging?

Reply:
Yes, repeat this line in this source code file:
az_span const _az_auth_header_name = AZ_SPAN_LITERAL_FROM_STR("authorization");
Or, just hard-code it in line 113.

Reply:
What is not to like? The values are kept in sync. Compilation error if _az_auth_header_name gets removed.